python/docs/source/reference/pyspark.mllib.rst

```diff
@@ -216,6 +216,8 @@ Statistics
     ChiSqTestResult
     MultivariateGaussian
     KernelDensity
+    ChiSqTestResult
+    KolmogorovSmirnovTestResult
```

Comment:
These two are returned by public methods so I believe that it makes sense to have documentation entries.